src/Common/Models/User.cs

```diff
@@ -7,13 +7,12 @@ public class User
 {
     public string Id { get; set; }
     public string Username { get; set; }
-    [Required]
-    public string Email { get; set; }
-    [Required]
-    public string Loyalty { get; set; }
-    [Required]
-    public string Name { get; set; }
+    [Required] public string Email { get; set; }
+    [Required] public string Loyalty { get; set; }
+    [Required] public string Name { get; set; }
     public List<Book> BooksRead { get; set; }
+    public string? ProfilePictureUrl { get; set; }
```

Comment:
Could maybe be a little more clear as a variable name as it could help indicate what this URL is for.